src/css/skins/seven.less

```diff
@@ -1,21 +1,20 @@
 @import "../imports/mixins";
 
 .jw-skin-seven {
-    @active-color: #FF0046;
-    @inactive-color: #fff;
+    @active-color: #ffffff;
+    @inactive-color: rgb(153, 153, 153);
     
     @cc-inactive: @inactive-color;
     @cc-active: @active-color;
 
     @rail-height: .2em;
 
     .jw-background-color {
-        background: #000;
+        background: rgba(37, 37, 37, 0.7); 
```

Comment:
This makes it very difficult to read the white current time over a light video (see big buck). If you are going to add transparency keep the color black so that text is always easy to see.

http://player-pr-test-jenkins.longtailvideo.com/builds/1867/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=skins_related_sharing